packages/bot/src/transformers/invite.ts

```diff
@@ -20,6 +21,24 @@ export function transformInvite(bot: Bot, payload: DiscordInviteCreate) {
   if (props.temporary && payload.temporary) invite.temporary = payload.temporary
   if (props.uses && payload.uses) invite.uses = payload.uses
 
+  if (props.approximateMemberCount && payload.approximate_member_count) {
+    invite.approximateMemberCount = payload.approximate_member_count
+  }
+
+  if (props.approximatePresenceCount && payload.approximate_presence_count) {
+    invite.approximatePresenceCount = payload.approximate_presence_count
+  }
+  if (props.guildScheduledEvent && payload.guild_scheduled_event) {
+    invite.guildScheduledEvent = payload.guild_scheduled_event ? bot.transformers.scheduledEvent(bot, payload.guild_scheduled_event) : undefined
+  }
+  if (props.stageInstance && payload.guild?.id && payload.stage_instance) {
+    invite.stageInstance = payload.stage_instance
+      ? bot.transformers.inviteStageInstance(bot, { ...payload.stage_instance, guildId: <BigString>invite.guildId })
+      : undefined
+  }
+  if (props.expiresAt) {
+    invite.expiresAt = payload.expires_at ? Date.parse(payload.expires_at) : undefined
```

Comment:
the payload.expires_at should be checked in the condition and then this ternary is not needed